### ISSUE TITLE
City state gets conquered with 1 turn to end barbarian Horde quest

### ISSUE THREAD

**Issue by Hugo-Pratas** [LoneGazebo/Community-Patch-DLL#10724]
_2. DX 11

    List of Other Mods
    InfoAddict, fast animations, furious turns, historical religions, squads for vp, 4uc

    Describe the Issue
    I don't know if this is intended but city state gets conquered with 1 turn to end barbarian Horde quest.

![imagem](https://github.com/LoneGazebo/Community-Patch-DLL/assets/90699949/53c7cb8c-6924-4a9b-a5ef-6bffab90611b)
legend: turn 139

![imagem](https://github.com/LoneGazebo/Community-Patch-DLL/assets/90699949/4da059ed-e457-4d5f-8801-31a6f35330f8)
legend: turn 140
[Logs.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/14559000/Logs.zip)

**Comment by tomekpa** [comment]
Yeah, happend to me multiple times also, my assumption, was, that if nobody help them, they will be conquered no matter what.
I noticed, its not a matter of conquering, but just flipping side to barbarians. Some strange mechanics i didnt understood, but maybe it is some bug.
Note, thatthere is less citizens, but city itself is healthy.

**Comment by Hugo-Pratas** [comment]
> Yeah, happend to me multiple times also, my assumption, was, that if nobody help them, they will be conquered no matter what. I noticed, its not a matter of conquering, but just flipping side to barbarians. Some strange mechanics i didnt understood, but maybe it is some bug. Note, that there is less citizens, but city itself is healthy. This case is even werider, since you have some units near, and you are ally. Just wild quess, you need to kill at least 1 barb last wave..

I actually killed just one barbarian the turn before, didn't manage to kill them all in time before the countdown. I also helped in the previous waves.
